dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/data/privacy/PrivacyConsentProvider.kt

```diff
@@ -0,0 +1,53 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.core.internal.data.privacy
+
+import java.util.LinkedList
+
+internal class PrivacyConsentProvider(consent: Consent = Consent.PENDING) :
+    ConsentProvider {
+
+    private val callbacks: LinkedList<ConsentProviderCallback> = LinkedList()
+
+    @Volatile
+    private var lastConsent: Consent
```

Comment:
Likewise you could just name that `consent` (and rename the constructor param `initialConsent`)